<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employee Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="mainTab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Select Employee</string>
     </attribute>
     <widget class="QTableWidget" name="EmployeeSelector">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>951</width>
        <height>501</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>741</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>Emp. #</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Seniority</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Phone #</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cell #</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Emergency Contact</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="refreshButton">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>590</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Employee:</string>
      </property>
     </widget>
     <widget class="QLabel" name="currentEmployeeLabel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>691</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
     <widget class="QPushButton" name="newEmployeeButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>590</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Add New</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Data</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Discipline</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>360</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Training</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>New</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>240</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Old</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ndviewbutton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>View</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ndupbutton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
     <widget class="QPushButton" name="odviewbutton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>230</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>View</string>
      </property>
     </widget>
     <widget class="QPushButton" name="odupbutton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>230</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
     <widget class="QPushButton" name="trainviewbutton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>440</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>View</string>
      </property>
     </widget>
     <widget class="QPushButton" name="trainupbutton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>440</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
     <widget class="QLabel" name="errorMessage">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>500</y>
        <width>401</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="ndimage">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>111</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>a</string>
      </property>
     </widget>
     <widget class="QLabel" name="odimage">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>241</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>a</string>
      </property>
     </widget>
     <widget class="QLabel" name="trainingimage">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>451</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>a</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Vacation</string>
     </attribute>
     <widget class="QTabWidget" name="VacationViewAllTab">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1021</width>
        <height>661</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Edit</string>
       </attribute>
       <widget class="QCalendarWidget" name="VacationCalendar">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>631</width>
          <height>461</height>
         </rect>
        </property>
       </widget>
       <widget class="QTabWidget" name="vacationTab">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>40</y>
          <width>321</width>
          <height>501</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Current Employee</string>
         </attribute>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>191</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Current Employee:</string>
          </property>
         </widget>
         <widget class="QLabel" name="vacationEmployeeName">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>261</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>None</string>
          </property>
         </widget>
         <widget class="QListWidget" name="vacationListWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>261</width>
            <height>311</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="vacationSaveButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>390</y>
            <width>261</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>Save Changes</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_8">
         <attribute name="title">
          <string>All Employees</string>
         </attribute>
         <widget class="QTextBrowser" name="vacationDayList">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>311</width>
            <height>461</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>View All</string>
       </attribute>
       <widget class="QCalendarWidget" name="Month1">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>0</y>
          <width>441</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QCalendarWidget" name="Month3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>220</y>
          <width>441</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QCalendarWidget" name="Month2">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>0</y>
          <width>441</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QCalendarWidget" name="Month4">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>220</y>
          <width>441</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="monthButtons1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>41</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="monthButtons2">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>470</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
       <widget class="QPushButton" name="monthButtons3">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>210</y>
          <width>41</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
